### DISCUSSION TITLE
Change "Ok" button for --dialog-on-download to "Dismiss"

### DISCUSSION THREAD

**Discussion by josh-ricketts-enable** [grahampugh/erase-install#509] [Ideas]
I wanted to see if would be possible to consider changing the text for the button on the **--dialog-on-download** window from **OK** to **Dismiss** or potentially something that makes it more apparent that clicking the button simply closes the window, while the process continues in the background. 

Perhaps this would just be a note beneath the main message saying something like:

_"NOTE: selecting 'OK' will close this window while this process continues running in the background"_

Thank you!

**Comment by grahampugh** [comment]
Hi, it's not a bad idea, but whatever message we add will need to be translated into 7 languages. I'll see if I can convert this discussion into a feature request and get some help with the translations.
